cropper/app/controllers/Application.scala

```diff
@@ -34,6 +34,9 @@ object Application extends Controller with ArgoHelpers {
 
   val mediaApiKey = keyStore.findKey("cropper").getOrElse(throw new Error("Missing cropper API key in key bucket"))
 
+  val parseExportRequest = parse.json(ExportRequest.readExportRequest)
```

Comment:
@jamesgorrie This seems like a much nicer way to parse input than the former forms imho.